python/dataproc_templates/gcs/README.md

```diff
@@ -0,0 +1,21 @@
+## GCS To BigQuery
+
+General Execution:
+
+```
+export GCP_PROJECT=<project_id> 
+export SUBNET=<region> 
+export JARS="gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar" 
+export GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> 
+export REGION=<region>
+
+./bin/start.sh \
+-- --template=GCSTOBIGQUERY \
+    --gcs.bigquery.input.format="<csv|parquet|avro>" \
```

Comment:
why is json not supported?

Reply:
I see no reason why it should not be supported, so I went ahead and added it.

One thing to note is that JSON is not supported in the Java version of this template. Should we consider adding it as well?

Reply:
great!
Per the java template, if that is the case, I would recommend creating a separate issue for that.